Makefile

```diff
@@ -128,7 +128,8 @@ ifneq ($(strip $(MG_MODULES)),)
 	$(call php, drush migrate:import --all --group=migrate_generator_group)
 	$(call php, drush pmu $(MG_MODULES) -y)
 	@echo "Deleting configs created by migrate_generator..." # https://dgo.to/3205125
-	for i in $(shell docker-compose exec --user $(CUID):$(CGID) php drush config:status --state=Any --format=list | grep -E 'migrate_plus.migration.migrate_generator|migrate_plus.migration_group.migrate_generator_group') ; do $(call php, drush config:delete -y $$i) ; done
+	$(eval MIGRATE_CONFIGS = $(shell docker-compose exec --user $(CUID):$(CGID) php drush config:status --state=Any --format=list | grep -E 'migrate_plus.migration.migrate_generator|migrate_plus.migration_group.migrate_generator_group'))
```

Comment:
Please use `$(call php` instead of docker compose

Reply:
It fails if I try assign `$(call php` result to a variable.

Reply:
There is no integrity in this solution, as migrate_generator provide configs, and after that Makefile try to deal and clean it. Nobody will find this place in case of issues.

Reply:
Also config dependencies are not taken into account, in case they exists, and they may exists in some specific cases.